### PR TITLE
Create DifficultyCommand.py

### PR DESCRIPTION
**Pull request by TheVeryStarlk**

### CODE REVIEW

File: src/podrum/command/vanilla/DifficultyCommand.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+*
+* Licensed under the Apache License, Version 2.0 (the "License")
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+"""
+
+from podrum.command.Command import Command
+from podrum.command import CommandManager
+
+class HelpCommand(Command):
```
Comment (HunterAzar) [inline src/podrum/command/vanilla/DifficultyCommand.py:16]: Change name of this Class to for example
```suggestion
class DifficultyCommand(Command):
```
beacuse class HelpCommand arleady exist.

File: src/podrum/command/vanilla/DifficultyCommand.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+"""
```
Comment (HunterAzar) [inline src/podrum/command/vanilla/DifficultyCommand.py:1]: For now we are focused to make joinable version.

**Review by HunterAzar — CHANGES_REQUESTED** [review]

File: src/podrum/command/vanilla/DifficultyCommand.py (hunk 1 above)
Comment (TheVeryStarlk) [reply src/podrum/command/vanilla/DifficultyCommand.py:16]: Opsie totally forgot to do that.

**Comment by Duerocraft** [comment]
in line 24 and blow code is like this "mode = int(args[1] == 0)"
code will break if args[1] is a string

**Comment by TheVeryStarlk** [comment]
> 
> 
> in line 24 and blow code is like this "mode = int(args[1] == 0)"
> code will break if args[1] is a string

It is converting mode to an integer

**Comment by MFDGaming** [comment]
@StarlkYT if you put like words as arguments it will either crash or just misbehave

**Comment by TheVeryStarlk** [comment]
> 
> 
> @StarlkYT if you put like words as arguments it will either crash or just misbehave

The else should prevent that?

**Comment by Duerocraft** [comment]
> 
> 
> > in line 24 and blow code is like this "mode = int(args[1] == 0)"
> > code will break if args[1] is a string
> 
> It is converting mode to an integer

if args[1] == 0 or "peaceful".lower():
    mode = int(args[1] == 0)

in this if input was int it will work correctly but if input was a string it always returns False

**Comment by TheVeryStarlk** [comment]
> 
> 
> > > in line 24 and blow code is like this "mode = int(args[1] == 0)"
> > > code will break if args[1] is a string
> > 
> > 
> > It is converting mode to an integer
> 
> if args[1] == 0 or "peaceful".lower():
> mode = int(args[1] == 0)
> 
> in this if input was int it will work correctly but if input was a string it always returns False

Indeed, I was told to do that by Nougator
I originally created a function that converts those keywords to integer.

Read converstation:
https://discord.com/channels/705730982338101280/715232125745627219/766664165787041792

**Pull request closed without merging**